nemo/collections/tts/modules/waveglow.py

```diff
@@ -85,51 +85,62 @@ def __init__(
             )
         self.n_remaining_channels = n_remaining_channels
 
-    @typecheck()
+    @typecheck(
+        input_types={
+            "spec": NeuralType(('B', 'D', 'T'), MelSpectrogramType()),
+            "z": NeuralType(('B', 'D', 'T'), MelSpectrogramType(), optional=True),
+            "sigma": NeuralType(optional=True),
+        },
+        output_types={"audio": NeuralType(('B', 'T'), AudioSignal())},
+    )
+    def infer(self, spec, z=None, sigma=1):
```

Comment:
@borisfom, I changed symbolic to infer, and added sigma to the function signature. Otherwise, this PR looks good to me. Let's wait for CI and merge?